src/wallet/wallet2.cpp

```diff
@@ -13297,6 +13391,14 @@ cryptonote::blobdata wallet2::export_multisig()
     // if we have 2/4 wallet with signers: A, B, C, D and A is a transaction creator it will need to pick up 1 signer from 3 wallets left.
     // That means counting combinations for excluding 2-of-3 wallets (k = total signers count - threshold, n = total signers count - 1).
     size_t nlr = tools::combinations_count(m_multisig_signers.size() - m_multisig_threshold, m_multisig_signers.size() - 1);
+
+    // 'td.m_multisig_k' is an expansion of [{alpha_0, alpha_1, ...}, {alpha_0, alpha_1, ...}, {alpha_0, alpha_1, ...}],
+    //    - each '{alpha_0, alpha_1, ...}' tuple can be used for one tx attempt using this 'transfer' (i.e. output)
```

Comment:
This comment doesn't seem to have been properly communicated. Nonces shouldn't be reused at all, not just for this output, as despite having a unique private key per output the factors needed to undo the unique key's derivation are known by the other members of the signing group.

Reply:
Ok updated the comments, let me know if it's more clear.

Reply:
LGTM